author-config.csv

```diff
@@ -1,9 +1,9 @@
 Repository's Location,Branch,Author's GitHub ID,Author's Email,Author's Display Name,Author's Git Author Name,Ignore Glob List
-https://github.com/reposense/testrepo-Beta.git,master,nbriannl,,Nbr,,
-https://github.com/reposense/testrepo-Beta.git,master,zacharytang,zacharytang.tc@gmail.com,Zac,,src/main**
-https://github.com/reposense/testrepo-Beta.git,master,April0616,,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting,
-https://github.com/reposense/testrepo-Beta.git,master,CindyTsai1,,Cin,YuHsuan,src/test**
-https://github.com/reposense/testrepo-Beta.git,add-config-json,nbriannl,,Nbr,,
-https://github.com/reposense/testrepo-Beta.git,add-config-json,zacharytang,zacharytang.tc@gmail.com,Zac,,src/main**
-https://github.com/reposense/testrepo-Beta.git,add-config-json,April0616,,Fan,LAPTOP-7KFM2KSP\User;Fan Yuting,
-https://github.com/reposense/testrepo-Beta.git,add-config-json,CindyTsai1,,Cin,YuHsuan,src/test**
+https://github.com/reposense/testrepo-Beta.git,master,nbriannl,,Labayna Neil Brian Narido,,
```

Comment:
I don't think it is a good idea to use one's full name who is not in our team. Please create a new test branch and add related test cases (CsvParser).

Reply:
My bad for this. But for the default `author-config.csv` file, will it be good if we still modify the display names to be more practical (but using random names instead)?

Reply:
I am not sure, my idea is just to leave it there. Can let the seniors decide.

Reply:
Ok. I'll revert changes to `author-config.csv` to the original state.